gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineCheckstyle.groovy

```diff
@@ -31,6 +31,7 @@ class BaselineCheckstyle extends AbstractBaselinePlugin {
     void apply(Project project) {
         this.project = project
 
+		project.rootProject.apply BaselineConfig
```

Comment:
I'm not sure this is kosher, but can't think of another way. In addition to adding this plugin to the root project, I think I need to require each project's checkstyle to depend on `:baselineUpdateConfig` (which should skip once it runs once, per changes)